Makefile

```diff
@@ -46,6 +46,11 @@ upgradebuild:
 upgradesetuptools:
 	python -m pip install --upgrade setuptools
 
+.PHONY: checkdeps
+checkdeps:
+	$(eval allow_list='numpy|pandas|featuretools|psycopg2|sqlalchemy|PyMySQL|snowflake-sqlalchemy[pandas]')
+	pip freeze | grep -v "github.com" | grep -E $(allow_list) > $(OUTPUT_PATH)
```

Comment:
```suggestion
	pip freeze | grep -v "alteryx/featuretools_sql" | grep -E $(allow_list) > $(OUTPUT_PATH)
```

Reply:
changed to `alteryx/featuretools*` to handle both featuretools and featuretools_sql lines

Reply:
Wait this isn't meant to be a catch all. It should be repo specific.